packages/core/src/defer/triggering.ts

```diff
@@ -433,6 +459,32 @@ export async function triggerHydrationFromBlockName(
   );
 }
 
+export function deferBlockHasErrored(lView: LView, tNode: TNode): boolean {
+  return getLDeferBlockDetails(lView, tNode)[DEFER_BLOCK_STATE] === DeferBlockState.Error;
+}
+
+function cleanupParentContainer(
+  i: number,
```

Comment:
nit:
```suggestion
function cleanupParentContainer(
  currentBlockIdx: number,
```
(I'd also propose adding a JsDoc comment to this function)